tox.ini

```diff
@@ -22,3 +22,4 @@ passenv =
     prawtest_test_subreddit
     prawtest_username
     prawtest_user_agent
+    prawtest_refresh_token
```

Comment:
Can you add this in sorted order please?

Reply:
Understood.